src/config/storage.ts

```diff
@@ -0,0 +1,50 @@
+import { ExtensionContext } from "vscode";
```

Comment:
Consider using `import * as vscode from "vscode"` for uniformity

Reply:
In that case this should be defined as a convention going forward.